cmd/kubelet/app/server.go

```diff
@@ -604,9 +597,7 @@ func kubeconfigClientConfig(s *options.KubeletServer) (*restclient.Config, error
 // --require-kubeconfig=true, we attempt to load the default kubeconfig file.
 func createClientConfig(s *options.KubeletServer) (*restclient.Config, error) {
```

Comment:
createAPIServerClientConfig, createClientConfig, and kubeconfigClientConfig are all only called from one code path, not in standalone mode. most of this error checking is moot at this point. I'd consider inlining this in  would just flattening those in the main flow in run()